src/main/java/longtimenosee/model/policy/Commission.java

```diff
@@ -39,7 +41,18 @@ public Commission(String commission) {
      * Returns true if a given string is a valid commission.
      */
     public static boolean isValidCommission(String test) {
-        return test.matches(VALIDATION_REGEX);
+        if (!test.matches(VALIDATION_REGEX)) {
+            return false;
+        }
+
+        float[] percentages = parseCommission(test);
+        for (float percentage: percentages) {
+            if (percentage < 0 || percentage > 100) {
+                return false;
```

Comment:
Performed light testing for this part. Just a side question, do you intend to allow decimal commissions? 

![image](https://user-images.githubusercontent.com/97422591/199455775-7f613dee-3ccb-4d85-a2c7-98a6cf3a41c9.png)
(I see that you allow decimals here)

Possibility for a bug:
![image](https://user-images.githubusercontent.com/97422591/199456040-dab7c3a1-1112-4ac0-87e0-971f4b378a47.png)
(Maybe you wanna limit the decimal or eliminate decimals completely)